paper.tex

```diff
@@ -257,6 +257,29 @@ \subsection*{Population genetics model}
 over that interval, which can be a constant value or function defined by start
 and end sizes that must remain positive.
 
+\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+	I'm not making an attempt yet to revise that later content--I think other authors should
+	weigh in.}
+
+\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+
+Within a population, we implicitly assume that allele frequency dynamics can
+be described by the "Wright-Fisher" model.  Briefly, generations are non-overlapping (all parents
+reproduce and die simultaneously) and the expected frequency (at birth) of an allele currently at frequency $p_i$
+one generation in the future is $p_iw_i/\bar{w}$, where $w_i$ and $\bar{w}$ are the marginal and
+mean fitnesses, respectively, properly weighted according to ancestry proportions. In this framework,
+a forward-time simulation of finite populations is equivalent to multinomial sampling of allele frequencies each
+generation (\citet[][pp 29-31]{burger2000-ul}, \citet[][pp 179-181]{crowkimura1970})
+and a backwards-time (coalescent) simulation follows the approximations described in \cite[][chapter 3]{wakeley2008-hd}.
+Further, this model assumes "soft" selection \citep{christiansen1975hard}, meaning that the dynamics of population sizes
+changes are \textit{independent} of the details of inidvidual fitnesses.
+Numerous examples of both types of simulators using this model currenly exit (REFS), each with their own unique method
+of specifying the demography of a simulation.
```

Comment:
Do you mean "specifying the details of selection in a simulation"?

Reply:
I can go either way on the wording here.